pkg/api/defaults-kubelet.go

```diff
@@ -55,7 +70,6 @@ func (cs *ContainerService) setKubeletConfig(isUpgrade bool) {
 	staticWindowsKubeletConfig["--cgroups-per-qos"] = "false"
 	staticWindowsKubeletConfig["--enforce-node-allocatable"] = "\"\"\"\""
 	staticWindowsKubeletConfig["--system-reserved"] = "memory=2Gi"
-	staticWindowsKubeletConfig["--client-ca-file"] = "c:\\k\\ca.crt"
```

Comment:
was this a bug? ie. previous behavior was to always override `--client-ca-file` to ` "c:\\k\\ca.crt"` for Windows, now behavior is to only set `--client-ca-file` to ` "c:\\k\\ca.crt"` if `EnableSecureKubelet` is true for Windows.

Reply:
Yes, the `--client-ca-file` has always been delivered in concert w/ the `EnableSecureKubelet` configuration option since the beggining:

Reference:

https://github.com/Azure/acs-engine/pull/2050